packages/babel-plugin/src/styled/__tests__/behaviour.test.tsx

```diff
@@ -417,14 +417,41 @@ describe('styled component behaviour', () => {
     );
   });
 
+  it('should apply conditional CSS with nested ternary operators', () => {
+    const actual = transform(`
+      import { styled } from '@compiled/react';
+
+      const Component = styled.button\`
+        color: \${(props) => (props.isPrimary ? props.isDisabled ? 'black' : 'blue' : 'red')};
+        font-size: 30px;
+        border: 2px solid blue;
+        padding: 8px;
+      \`;
+    `);
+
+    expect(actual).toIncludeMultiple([
+      '._19bvftgi{padding-left:8px}',
+      '._n3tdftgi{padding-bottom:8px}',
+      '._u5f3ftgi{padding-right:8px}',
+      '._ca0qftgi{padding-top:8px}',
+      '._19itlf8h{border:2px solid blue}',
+      '._1wyb1ul9{font-size:30px}',
+      '._syaz1xo0{color:var(--_13kr4bd)}',
+    ]);
+
+    expect(actual).toInclude(
```

Comment:
This `toInclude` can go inside `toIncludeMultiple`. I think we're trying to keep everything in one assertion if possible.